features/taxation/managing_tax_rates/adding_tax_rate.feature

```diff
@@ -0,0 +1,24 @@
+@managing_tax_rates
+Feature: Adding a new tax rate
+    In order to apply different taxes rates to various tax categories
+    As an Administrator
+    I want to add a new tax rate to the registry
+
+    Background:
+        Given there is "EU" zone containing all members of European Union
+        And the store has a tax category "Food and Beverage" with a code "food_and_beverage"
+        And the store has a default calculator defined
+        And I am logged in as an administrator
+
+    @todo
+    Scenario: Adding a new tax rate
+        Given I want to create a new tax rate
+        When I specify its code as "EUROPEAN_UNION_SALES_TAX"
+        And I name it "European Union Sales Tax"
+        And it is defined for a EU zone
```

Comment:
Anyway, we use "I specify its code", "I name it" but the "it is defined", "it defines", looks inconsistent.

Shouldn't it be "I define it for the EU zone", "I make it applicable for "Food and Beverage" tax category", etc.?